packages/react/src/components/Forms/LabelledBox/index.tsx

```diff
@@ -13,12 +15,20 @@ export const LabelledBox = ({
   labelAsPlaceholder,
   htmlFor,
   labelId,
+  showLabel = true,
```

Comment:
Should be `hideLabel` what do you think ?

Reply:
Lezgo